### PR TITLE
Store idea.log from UI tests

### PR DESCRIPTION
**Pull request by mkondratek**
DEPENDS ON https://github.com/VirtusLab/ide-probe/pull/266

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -77,74 +77,68 @@ jobs:
           name: Start Gradle daemon
           command: ./gradlew
 
+#      - run:
```
Comment (mkondratek) [inline .circleci/config.yml:80]: - [ ] to be reverted

File: .circleci/config.yml [hunk 2]
```diff
@@ -167,11 +161,16 @@ jobs:
             if ! ( [ -d ~/.ideprobe-uitests/artifacts ] && [ -n "$(ls -A ~/.ideprobe-uitests/artifacts)" ] ); then exit 0; fi
             for f in $(ls -d ~/.ideprobe-uitests/artifacts/*/) ; do cd  $f ; zip -r "../$(basename ${f}).zip" . ; done
             mkdir -p ~/artifacts/zip
+            mkdir -p ~/artifacts/logs
             find ~/.ideprobe-uitests/artifacts -name '*.zip' | xargs cp --target-directory ~/artifacts/zip
+            find ~/.ideprobe-uitests/idea-logs/intellij-instance-*/logs/idea.log | xargs cp --target-directory ~/artifacts/logs
```
[the remark this answers is not in the document]
Comment (PawelLipski) [reply .circleci/config.yml:166]: IIRC the `intellij-instance-*` path segment includes IDE version

File: .circleci/config.yml [hunk 3]
```diff
@@ -170,11 +170,19 @@ jobs:
...
             find ~/.ideprobe-uitests/artifacts -name '*.zip' | xargs cp --target-directory ~/artifacts/zip
+            for f in $(find ~/.ideprobe-uitests/idea-logs/intellij-instance-*/logs/idea.log) ; do
+              target="~/artifacts/logs/$(echo $f | sed 's/.*\/\(intellij-instance-.*\)\/logs\/idea.log/\1.log/')"
+              eval "cp $f $target"
+            done
           when: on_fail
       - store_artifacts:
           path: ~/artifacts/zip
           destination: .
+      - store_artifacts:
```
Comment (mkondratek) [inline .circleci/config.yml:183]: @PawelLipski does it make sense to include whole `~/artifacts` in the artifacts?

Comment (PawelLipski) [reply .circleci/config.yml:183]: Hmmm what else can be there?

Comment (PawelLipski) [reply .circleci/config.yml:183]: Other than logs? Also screenshots right?

Comment (mkondratek) [reply .circleci/config.yml]: yup, here is what we have if ci failes:
https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/6488/workflows/76db5f9d-0931-497e-b91d-47f8ab8b1199/jobs/6866/artifacts
and here, when it passes:
https://app.circleci.com/pipelines/github/VirtusLab/git-machete-intellij-plugin/6489/workflows/dbbeb06d-248a-48c7-b133-225c0e02f86b/jobs/6867/artifacts

**Review by PawelLipski — APPROVED** [review]

**Pull request merged**